src/python/benchUtil.py

```diff
@@ -1536,12 +1539,15 @@ def getClassPath(checkout):
 
   # We use the jar file for core to leverage the MR JAR
   core_jar_file = None
+  jars_found = 0
   for filename in os.listdir('%s/lucene/core/build/libs' % path):
     if reCoreJar.match(filename) is not None:
       core_jar_file = '%s/lucene/core/build/libs/%s' % (path, filename)
```

Comment:
Uh oh, why is this path different from the above path!  Why are we doing this (differently!) in two places?

Reply:
This line in `getClassPath` line 1535:
```
  if not os.path.exists(os.path.join(path, 'build.gradle')):
    return getAntClassPath(checkout)
```
The top path is `getAntClassPath`, and the bottom is `getClassPath`

Reply:
OK this is actually dead-ish code now, but let's leave it since sometimes we do go back and (try to) run `luceneutil` on older Lucene releases.